active_learning_multiannotator/utils/model_training.py

```diff
@@ -0,0 +1,59 @@
+"""
+This file defines a function to fit a cross-validation model and produce out-of-sample
+predicted probabilites for examples in active learning.
+"""
+import numpy as np
+import sklearn
+from sklearn.model_selection import cross_val_predict
+from sklearn.model_selection import StratifiedKFold
+
+from cleanlab.internal.util import (
+    get_num_classes,
+    append_extra_datapoint,
+    train_val_split,
+)
+
+
+def fit_predict_proba(
+    model,
+    X,
+    labels,
```

Comment:
Probably want to clarify these are consensus labels, derived for a particular example by aggregating all collected  annotations into a single one. And the shape expected for `labels`